### PR TITLE
Fix #488: Remove Trackers + Script blocking from global stats view

### LINKED ISSUE #488: New tab doesn't increase count for Trackers blocked
### Description: 
New tab doesn't increase count for Trackers blocked

### Steps to Reproduce 
  1. Visit a script heavy site like youtube.com/slashdot.org/theverge.com
  2. Check blocked items on shields 
  3. Open a new tab, same blocked count is shown for ads while tracker protection is always 0 no matter how many sites you visit

**Actual result:** 
New tab doesn't increase count for Trackers blocked

**Expected result:**
Ad blocked count and trackers blocked count should be independently shown on new tab 

**Reproduces how often:** [Easily reproduced, Intermittent Issue]
Easy

**Brave Version:** 
1.7 (18.11.28.22)

**Device details:** 
All

**Website problems only:**
- did you check with Brave Shields down?
- did you check in Safari/Firefox (WkWebView-based browsers)?

### PR DESCRIPTION
**Pull request by kylehickinson**
- Also fixes ads + trackers count in domain-specific shield stats not including trackers count

## Pull Request Checklist

- [x] My patch has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!`
- [ ] I have updated the *Unit Tests* to cover new or changed functionality
- [ ] I have updated the *UI Tests* to cover new or changed functionality
- [x] I have made sure that localizable strings use `NSLocalizableString()`

## Screenshots

simulator screen shot - iphone x - 2018-11-30 at 10 24 11

## Notes for testing this patch

_None included_

### CODE REVIEW

**Comment by kylehickinson** [comment]
Added @danishjafri88, to check the localized string part

**Review by jhreis — COMMENTED** [review]
Waiting on copy approval.

**Review by jhreis — APPROVED** [review]

**Pull request merged**
